README.md

```diff
@@ -75,6 +75,15 @@ Extend real data from GitHub API with faked data based on extension IDL (you can
  * `--forward-headers`     Specify which headers should be forwarded to the proxied server
  * `--co`, `--cors-origin` CORS: Specify the origin for the Access-Control-Allow-Origin header
  * `-h`, `--help`          Show help
+ 
+When specifying the `[IDL file]` after the `--forward-headers` option you need to prefix it with `--` to clarify it's not another header. For example:
```

Comment:
I think `SDL` is a more popular term right now.
It looks like TSDL would be most correct one but I want to wait for official response:
https://github.com/facebook/graphql/pull/456
So it should be `SDL` for now.

Reply:
Alright.